pkgs/by-name/ca/calceph/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  cmake,
+  gfortran,
+}:
+stdenv.mkDerivation rec {
+  pname = "calceph";
+  version = "4.0.5";
+  src = fetchzip {
+    url = "https://www.imcce.fr/content/medias/recherche/equipes/asd/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-s42PQd2LEN+zwglNDjhUoofv+RdVyvpD3Pk0EaNAFSM=";
+  };
```

Let's get the source from their GitLab [imcce_calceph/calceph](https://gitlab.obspm.fr/imcce_calceph/calceph)

```suggestion
{
  lib,
  stdenv,
  fetchFromGitlab,
  cmake,
  gfortran,
}:
stdenv.mkDerivation rec {
  pname = "calceph";
  version = "4.0.5";
  src = fetchFromGitlab {
    domain = "https://gitlab.obspm.fr";
    owner = "imcce_calceph";
    repo = "calceph";
    tag = "calceph_${builtins.replaceStrings [ "." ] [ "_" ] version}";
    hash = "sha256-s42PQd2LEN+zwglNDjhUoofv+RdVyvpD3Pk0EaNAFSM=";
  };
```